server/src/jobs/impl/nut/process.cpp

```diff
@@ -38,7 +38,7 @@ Expected<void> Process::setupSnmp(const std::string& address, uint16_t port)
         m_process = std::unique_ptr<fty::Process>(new fty::Process(*path, {
             "-s", "discover",
             "-x", fmt::format("port={}", toconnect),
-            "-d", "1"
+            "-d", "2"
```

Comment:
Are we sure about the -d 2 ? We added it before and we had to remove it as the UI was timing out